### PR TITLE
Set `action` variable before it is used

### PR DESCRIPTION
**Pull request by MisinformedDNA**
**Task name**: AzureResourceManagerTemplateDeploymentV3

**Description**: `action` needs to be set before it is used

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) #13148 

**Checklist**:
- [X] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [ ] Checked that applied changes work as expected

### CODE REVIEW

**Comment by bishal-pdMSFT** [comment]
LGTM

@MisinformedDNA please bump up the task version. The help for that is specified in the PR checklist

File: Tasks/AzureResourceManagerTemplateDeploymentV3/task.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
     "author": "Microsoft Corporation",
     "version": {
         "Major": 3,
-        "Minor": 2,
+        "Minor": 172,
```
Comment (anuragc617) [inline Tasks/AzureResourceManagerTemplateDeploymentV3/task.json:17]: please update the patch version, bumped up version should be 3.2.7

Comment (MisinformedDNA) [reply Tasks/AzureResourceManagerTemplateDeploymentV3/task.json:17]: I'm confused how the versioning is supposed to work. My version was certainly wrong, but I'm not sure how to correlate increasing the patch number with [the instructions](https://github.com/microsoft/azure-pipelines-tasks/blob/master/docs/taskversionbumping.md):

  1. Check current Azure DevOps sprint
    - Sprint is 172
  2. If sprint number differs from current minor number - set it to current sprint number, set patch to 0
    - `172 != 2` so `minor = 172` and `patch = 0`

So then it should be `3.172.0`?

Comment (anuragc617) [reply Tasks/AzureResourceManagerTemplateDeploymentV3/task.json:17]: @MisinformedDNA Yes, you can use the 3.172.0. For AzureResourceManagerTemplateDeploymentV3 task, we use to increment the patch version for minor fix and minor version for some enhancements in the task and major for new features.

Comment (MisinformedDNA) [reply Tasks/AzureResourceManagerTemplateDeploymentV3/task.json]: Cool. Just wanted to understand. Fixed.

**Pull request merged**